doc/connectivity/bluetooth/api/mesh/lcd_srv.rst

```diff
@@ -0,0 +1,27 @@
+.. _bluetooth_mesh_lcd_srv:
+
+Large Composition Data Server
+#############################
+
+The Large Composition Data Server model is a foundation model defined by the Bluetooth
+mesh specification. The model is optional, and is enabled through the :kconfig:option:`CONFIG_BT_MESH_LARGE_COMP_DATA_SRV` option.
+
+The Large Composition Data Server model was introduced in the Bluetooth Mesh Protocol Specification version 1.1, and is used to support
+the functionality of exposing pages of Composition Data that do not fit in a Config Composition Data Status message and to expose metadata of the model instances.
+
+Metadata contains information about the models instantiated on the element. The structure member :c:member:`bt_mesh_models_metadata_entry` can be used to define a list of metadata entries.
+Entries should be created using the `BT_MESH_MODELS_METADATA_ENTRY` macro. Each entry contains the length and ID of the metadata, and a pointer to the raw data.
+If the list is empty, the helper macro `BT_MESH_MODELS_METADATA_NONE` can be used instead.
+The `BT_MESH_MODELS_METADATA_END` macro marks the end of the metadata list and must always be present.
```

Comment:
I somehow managed to miss this yesterday... Shouldn't these macros be written with :c:macro:`BT_MESH_MODELS_METADATA_ENTRY`, etc?

Reply:
These comments render strange sometimes. I am using ` in front and end of the name.